openlibrary/plugins/openlibrary/home.py

```diff
@@ -50,9 +49,10 @@ def get_homepage():
     )
     return dict(page)
 
-def get_cached_homepage():
+
+def get_cached_homepage(lang):
```

Comment:
Ooops! Meant this to be part of the previous changes.
```suggestion
def get_cached_homepage():
```